ui/Screen/Settings.svelte

```diff
@@ -2,52 +2,79 @@
   import {
     clear,
     clearCache,
+    fetchSeeds,
     settings,
     updateAppearance,
+    updatePeerConfig,
     updateRegistry,
   } from "../src/session.ts";
   import { networkOptions, themeOptions } from "../src/settings.ts";
 
-  import { Title, Text, Button } from "../DesignSystem/Primitive";
+  import { Button, Input, Text, Title } from "../DesignSystem/Primitive";
   import { SidebarLayout, SegmentedControl } from "../DesignSystem/Component";
 
+  let seeds = fetchSeeds();
+
   const updateNetwork = event =>
     updateRegistry({ ...$settings.registry, network: event.detail });
 
   const updateTheme = event =>
     updateAppearance({ ...$settings.appearance, theme: event.detail });
+
+  const updateSeeds = event => {
+    updatePeerConfig(event.target.value);
+  };
 </script>
 
 <style>
   .container {
     max-width: var(--content-max-width);
-    margin: 64px auto;
+    margin: 64px 100px;
     min-width: var(--content-min-width);
     padding: 0 var(--content-padding);
   }
+
   section header {
     margin: 16px 0 24px 0;
     border-bottom: 1px solid var(--color-foreground-level-3);
-    padding: 12px;
+    padding: 12px 0;
+    display: flex;
+    justify-content: space-between;
   }
+
+  section header a {
+    color: var(--color-secondary);
+  }
+
+  section header a:hover {
+    text-decoration: underline;
+  }
+
   .section-item {
-    padding: 0 12px;
     display: flex;
     justify-content: space-between;
     align-items: center;
     margin-bottom: 24px;
   }
+
   .action {
     display: flex;
     justify-content: space-between;
     align-items: center;
     margin-left: 16px;
   }
+
+  /* TODO(sos): implement responsive margins once we have screen-based layouts */
+  @media (max-width: 1000px) {
+    .container {
+      margin: 64px auto;
+    }
+  }
```

Comment:
Does this have to do with [this](https://github.com/radicle-dev/radicle-upstream/pull/629/files#diff-a2343ce5bd1bd043d92dd0abf58e632eR32)

Reply:
Yup - trying to make it look more like the spec, which I actually think looks nicer than the stretched version. I can remove it for now and we can do a responsive overhaul before the release. related to #577